src/main/java/com/aphyr/riemann/client/Callback.java

```diff
@@ -0,0 +1,7 @@
+package com.aphyr.riemann.client;
+
+public interface Callback<T> {
+
+  public void call(T value, Exception exception);
```

Comment:
I suggest splitting out the callback into a pair of interfaces: one for exceptions, one for standard returns.